Gemfile

```diff
@@ -84,6 +84,7 @@ group :development, :test do
   gem "pry"
```

Comment:
I think Tricia uses a vagrant dev environment, so I should keep pry-remote in for now.